src/consensus/params.h

```diff
@@ -7,6 +7,7 @@
 #define BITCOIN_CONSENSUS_PARAMS_H
 
 #include <uint256.h>
+
```

Comment:
nit

Reply:
Yes, there should be a newline between project-internal includes and project-external ones.

Reply:
Force pushed to remove the nit-fix, since it is being fixed in another pull already.